packages/ssr/src/incremental.rs

```diff
@@ -152,6 +152,7 @@ impl IncrementalRenderer {
             }
         }
         // check the file cache
+        #[cfg(not(target_arch = "wasm32"))]
```

Comment:
It looks like cloudflare workers supports wasi, tokio has [experimental support](https://docs.rs/tokio/latest/tokio/index.html#unstable-wasm-support) for wasi, and wasi has experimental support for fs operations. Does cloudflare not support fs, or does tokio not support wasi fs?

Reply:
Cloudflare appears to support WASI.

https://github.com/cloudflare/workers-wasi

However, I don't think actual Workers supports storage.

https://community.cloudflare.com/t/about-cloudflare-workers-filesystem/481642/2